x/gamm/keeper/pool_service_test.go

```diff
@@ -222,6 +241,147 @@ func (suite *KeeperTestSuite) TestCreateBalancerPool() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestInitializePool() {
+	params := suite.App.GAMMKeeper.GetParams(suite.Ctx)
+	testAccount := suite.TestAccs[0]
+
+	// get raw pool creation fee(s) as DecCoins
+	poolCreationFeeDecCoins := sdk.DecCoins{}
+	for _, coin := range params.PoolCreationFee {
+		poolCreationFeeDecCoins = poolCreationFeeDecCoins.Add(sdk.NewDecCoin(coin.Denom, coin.Amount))
+	}
+
+	_, err := balancer.NewBalancerPool(1, defaultPoolParams, defaultPoolAssets, defaultFutureGovernor, time.Now())
+	require.NoError(suite.T(), err)
+
+	_, err = stableswap.NewStableswapPool(1, defaultStableSwapPoolParams, defaultStableSwapPoolAssets, defaultScalingFactor, "", defaultFutureGovernor)
+	require.NoError(suite.T(), err)
```

Comment:
I don't think we need any of these since we create a pool for each test case

Reply:
I agree.

Reply:
Sweet, please remove this then, and then we can merge the PR! Everything else LGMT

Reply:
I have removed the redundant logic. Tks Roman